examples/light-switch-app/nrfconnect/boards/nrf52840dongle_nrf52840.conf

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
I don't think we are going to support dongle in this example, so this file is not needed.